src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -316,34 +347,25 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', function (c3Cha
     link: function (scope, element) {
       var setupDonutChartTitle = function () {
         $timeout(function () {
-          var donutChartTitle, bigText, smText;
+          var donutChartTitle, centerLabelText;
 
           donutChartTitle = element[0].querySelector('text.c3-chart-arcs-title');
-          if (scope.config.centerLabelFn) {
-            donutChartTitle.innerHTML = scope.config.centerLabelFn(scope);
-          } else if (scope.centerLabel === 'none') {
-            donutChartTitle.innerHTML = '';
+          if (!donutChartTitle) {
+            return;
+          }
+
+          centerLabelText = scope.getCenterLabelText(scope);
+
+          if (scope.config.centerLabelFn || scope.centerLabel === 'none') {
+            donutChartTitle.innerHTML = centerLabelText.bigText;
```

Comment:
That makes both the getCenterLabelText function and the code outside of it know how to handle the Fn return value and how to handle none. Setting to None means the getCenterLabelText function MUST return "" for bigText and outside must set innerHTML to bigText not put it within the tspan. Harder to read and harder to be sure none is being handled correctly. When using the Fn, smallText is still set to scope.config.units + ' Used' so the outside needs to know not to use smallText.